### ISSUE TITLE
k(5500 A) = 1 normalization is honored by some attenuation laws and not others (up to 0.29%)

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#1731]
## Summary

`tests/crossval/test_attenuation_physics.py::test_k_at_5500_equals_one` documents `k(5500 A) = 1.0` as "the normalization convention", and most attenuation laws honor it exactly. Analytic paper fits do not — they return the value their published polynomial gives at 5500 A, which is near but not equal to 1. The result is that `dust_tau_diff` / `dust_tau_bc` mean slightly different things depending on which curve is selected.

## Measured

```python
from tengri.components.dust.attenuation import resolve_dust_law
resolve_dust_law(name)(jnp.array([5500.0]))
```

| Law | k(5500 A) | Deviation |
|---|---|---|
| `smc`, `lmc`, `salim`, `conroy2010`, `kriek_conroy`, `wd01_mwrv31` | 1.000000 | exact |
| `calzetti` | 0.999479 | 0.05% |
| `noll09` | 0.999479 | 0.05% |
| `cardelli` | 0.998850 | 0.12% |
| `reddy15` | 0.997113 | 0.29% |

The split is not arbitrary: the exact-1.0 group is tabulated or explicitly renormalized, and the others evaluate a published fitting function at 5500 A. CCM89's optical/NIR polynomial is a fit, and it does not pass exactly through `a(x) + b(x)/R_V = 1` at `x = 1/0.55 = 1.818 um^-1`. Reproducing the paper faithfully and normalizing to unity at V are, for these curves, mutually exclusive.

## Why it matters

`tau_v` is defined against `k(V)`. If `k(5500) != 1`, then a fitted `dust_tau_diff` is an optical depth at a slightly different effective normalization, and switching `law_bc='salim'` to `law_bc='reddy15'` rescales the meaning of the same `tau` value by 0.29% with no warning. Small, but systematic, and it propagates into A_V and into any dust-corrected SFR.

It also means the test currently encodes a convention the codebase only partly follows. `cardelli` fails it at `atol=1e-3`; `reddy15` would fail harder but is not in that test's parametrize list, so nothing catches it.

## The decision (yours, not mine)

Three coherent options, and they are not equally good:

1. **Normalize every curve to `k(5500) = 1` at construction.** Makes `tau_v` mean one thing everywhere. Costs exact parity with CCM89/Calzetti/Reddy at the 0.05-0.29% level, which would show up in any bit-exact cross-code comparison.
2. **Keep the published values and drop the convention.** Faithful to the papers; requires documenting that `tau_v` is per-curve and adjusting the test to assert each curve's own published value with a citation.
3. **Normalize only where the source curve is itself defined relative to A_V**, and document the exceptions. Most faithful, most bookkeeping.

I have not changed any physics here — it is a convention call with numerical consequences for published results, and #1728 is already full of tests that were repaired by picking the convention the code happened to implement. Whichever way this goes, the test should assert it for **all** registered laws rather than the current subset, so the next curve added cannot quietly join the wrong group.

Found while repairing the crossval suite (#1728).
